ts/store/reducers/notifications/localScheduled.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Local scheduled notifications reducer
+ */
+
+import { getType } from "typesafe-actions";
+
+import {
+  clearLocalNotificationsScheduled,
+  updateLocalNotificationsScheduled
+} from "../../actions/notifications";
+import { Action } from "../../actions/types";
+import { GlobalState } from "../types";
+
+export type LocalScheduledState = Readonly<{
+  isToSchedule: boolean;
+}>;
+
+export function getInitialState(): LocalScheduledState {
+  return {
+    isToSchedule: true
+  };
+}
+
+const reducer = (
+  state: LocalScheduledState = getInitialState(),
+  action: Action
+): LocalScheduledState => {
+  switch (action.type) {
+    case getType(updateLocalNotificationsScheduled):
+      return { ...state, isToSchedule: action.payload };
+
+    case getType(clearLocalNotificationsScheduled):
+      return getInitialState();
+
+    default:
+      return state;
+  }
+};
+
+// Selectors
+export const notificationsLocalScheduledSelector = (state: GlobalState) => {
+  return state.notifications.scheduled;
```

Comment:
since you're using this selector just of `.isToSchedule`, you can move it here:

```suggestion
  return state.notifications.scheduled.isScheduled;
```

Reply:
I fixed it in the commit [3c874ae](https://github.com/teamdigitale/io-app/pull/992/commits/3c874aee998ce81853bd3bdc26d881f855192ef1)